### ISSUE TITLE
Windows telnet does not work

### ISSUE THREAD

**Issue by daniele77** [daniele77/cli#19]

**Comment by sdebionne** [comment]
Hi Daniele, could you be more specific? I have an issue with the server not 'echoing' input char on Windows and I am not sure if that is what you are referring to with this issue?

**Comment by daniele77** [comment]
@sdebionne you're right, when I opened the issue it was more a reminder for me than a real description. Sorry.

The server (i.e., the application that uses `cli` library) runs correctly on both Linux and windows machines.

However, I noticed that when you use the Windows telnet client to connect, it does not work.

So, if you're using the standard windows telnet client to connect remotely to the `cli` and you're experimenting any problem, it falls into this GitHub issue.

**Comment by sdebionne** [comment]
I see, thanks for clarifying. I use Putty so this is not my issue!
